package.json

```diff
@@ -32,7 +32,7 @@
     "lint-fix": "eslint . --ext .js,.ts --fix",
     "pre-commit": "yarn lint-staged",
     "pre-push": "yarn format-check",
-    "postinstall": "lerna bootstrap && yarn build && lerna link && npm run check-clean-workspace-after-install && opencollective-postinstall",
+    "postinstall": "lerna bootstrap && yarn build && lerna link && npm run check-clean-workspace-after-install",
```

Comment:
question:
`npm run check-clean-workspace-after-install` is pretty annoying here.
The intention of it is to ensure there aren't any lockfile changes in the CI

Do we want to move this to the azure script?

Reply:
i don't like it to, its fails always when you have local changes